app/assets/javascripts/sufia/autocomplete/language.es6

```diff
@@ -0,0 +1,25 @@
+export class Language {
```

Comment:
@jcoyn The 3 classes Language, Subject &  look to be exact copies.  Was that intentional?